data_steward/validation/main.py

```diff
@@ -16,7 +16,9 @@
 # Third party imports
 import dateutil
 from flask import Flask
+from google.cloud.exceptions import GoogleCloudError
 from googleapiclient.errors import HttpError
+
```

Comment:
It would be nice to get rid of all HttpError usages and remove this completely, but that can be in a later ticket. Btw, you don't have to add blank lines within import blocks. Just make sure they are grouped together if coming from the same package/subpackage.

Reply:
I forgot to remove the blank line. I think I left there by mistake when testing different exceptions.